src/PushAdmin.php

```diff
@@ -4,6 +4,8 @@
 class PushAdmin {
 
     private $ably;
+    public $deviceRegistrations;
+    public $channelSubscriptions;
```

Comment:
This is related to the spec -> https://docs.ably.io/client-lib-development-guide/features/#RSH1
Can we create a separate PR for this, that includes proper tests.

Reply:
The main purpose of this PR is to fix exactly this. The PHP 8.2 build covers this.

Reply:
Do we have tests covering this?

Reply:
Looking into this. For some reason the PHP 8.2 isn't detecting the calls to undefined properties.

Reply:
I'm baffled at why the tests aren't failing on my GHA build. I'm testing locally and I'm getting the deprecation warnings for PHP 8.2.

Reply:
I finally got the deprecations to show up. I needed to add `ini-values: error_reporting=E_ALL` to the PHP setup script in the tests and `convertDeprecationsToExceptions="true"` to the `phpunit.xml` file so they make the build fail. So now they'll fail on PHP 8.2:

<img width="785" alt="Screenshot 2022-09-05 at 14 35 23" src="https://user-images.githubusercontent.com/594614/188450876-b6f74b6e-355a-4750-9ff7-4663f1a52d50.png">